schemas/context/profile-optinout.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+		"Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+		"This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+		"you may not use this file except in compliance with the License. You may obtain a copy",
+		"of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-optinout",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile Opt In/Out Signals",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "title": "Profile Opt Out",
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Profile Optout Mixin",
+  "definitions": {
+	"profile-optinout": {
+		"type": "object",
+			"properties": {
+				"xdm:privacyOptouts": {
+					"title": "Privacy Opt Outs",
+					"description": "Encapsulates different optout types for Data Privacy",
+					"type": "array",
+					"items": {
+					"$ref": "#/definitions/optout"
+					}
+				},
+				"xdm:channelOptouts": {
+					"title": "Channel Opt Outs",
+					"$ref": "https://ns.adobe.com/xdm/context/optintout.schema.json"
+				}
+			}
+    },
+    "optout": {
+		"type": "object",
+		"properties": {
+			"xdm:optoutType": {
+				"title": "Optout Type",
+				"type": "string",
+				"description": "Type of optout",
+				"meta:enum": {
+					"generalOptout": "generalOptout",
```

Comment:
`meta:enum` uses the format `<enum id>: "<Display Name>"

The second value in this schema should be human readable for example:

```
"meta:enum": {
	"generalOptout": "generalOptout",
	"salesSharingOptout": "salesSharingOptout"
}
```